tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -1055,11 +1053,15 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 				vmUUID, pv.Spec.CSI.VolumeHandle))
 		defer func() {
 			testCleanUpUtil(ctx, restConfig, cnsRegisterVolume, namespace, pvc.Name, pv.Name)
-			if isQuotaValidationSupported {
-				validateQuotaUsageAfterCleanUp(ctx, restConfig, storagePolicyName, namespace, pvcUsage,
-					volExtensionName, diskSizeInMbstr, totalQuotaUsedAfter, storagePolicyQuotaAfter,
-					storagePolicyUsageAfter, false)
-			}
+			//Validates PVC quota in both StoragePolicyQuota and StoragePolicyUsage CR
```

Comment:
We need to add if condition for isQuotaValidationSupported to only check for VC version where storage quota feature is supported.